gsms.py

```diff
@@ -153,6 +157,176 @@ def get_win_path(folder_id: str) -> str:
     # Free memory used by pointer
     _CoTaskMemFree(path_pointer)
 
+    print(path)
+
+    return path
+
+
+def copy_image(src_image: str, dst_image: str) -> None:
+    """
+    Copies an image from src_image to dst_image if the dst is empty or different
+
+    Parameters
+    ----------
+    src_image: str
```

Comment:
for all parameters in docstrings, this is the format
```rst
<name> : <type>
    A sentence, or two, describing the parameter. It should end with a period or proper punctuation!
```

P.S. These docstrings are reStructuredText (rst).

Reply:
I swear we had this last PR as well... how could I forget